README.md

```diff
@@ -42,6 +43,15 @@ or
 npm install
 ```
 
+## Environment variables
+
+To connect your Meilisearch instance and retrieve the movies two environment variables should be provided:
+
+- HOST
+- API_KEY
```

Comment:
Since it's the specific host and apiKey of Meilisearch and not this app, I would add a reference to MeiliSearch to the variable name, otherwise, it might be confusing as to what it refers to. Both in the .env or in the local environment variables (export..).

But I understand it implies changes in a lot of different files! What do you think? Maybe an issue? Or not if you prefer not.

Reply:
Yes let's open a different issue for that 👍

Reply:
Here it is: #373